Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
> I think the ternary should only be used in expressions

True, it is more familiar in that usage, and the ternary looks weird if you're not used to seeing it "free-floating" without taking the return value. So, `if ... else` is just fine.